### DISCUSSION TITLE
Some(a lot of) local beatmap disappeared after the latest update.

### DISCUSSION THREAD

**Discussion by IkaWaAyuMu** [ppy/osu#16654] [Q&A] [answered]
After I updated lazer to the latest version, It shown migrating to realm (that scary black screen and few things). Then entered the game. Then I checked something ... a lot of my map disappeared (I can't count but a lot).

I tried restart the game no scary black screen appeared again so I assume that I cant migrate again.

Is the disappeared beatmap still in my local storage? Can I get the map back?

(I restarted the game so the log is gone :( )

**Comment by peppy** [comment, marked as answer]
Please compress and attach all the files in the "backups" directory.

**Reply by IkaWaAyuMu** [reply]
[backups.zip](https://github.com/ppy/osu/files/7951551/backups.zip)
This one.

**Reply by peppy** [reply]
Here's a fixed post-realm database. Please close osu! and replace the client.realm file in your main osu! directory with this one:
[client.realm.zip](https://github.com/ppy/osu/files/7952044/client.realm.zip)

**Reply by IkaWaAyuMu** [reply]
> Here's a fixed post-realm database. Please close osu! and replace the client.realm file in your main osu! directory with this one: [client.realm.zip](https://github.com/ppy/osu/files/7952044/client.realm.zip)

Other than mute shortcut that changed to `Ctrl+F4` (I changed back.) everything else is fine.

**Reply by IkaWaAyuMu** [reply]
Oh I've just realized after play that my skin is gone :sob:.

**Reply by peppy** [reply]
In your backup folder, there was no `client.realm` file. Do you still have the backup of this? If so I can recover your skins.
